### PR TITLE
[TECHNICAL] Rely on `resharing` capability

### PR DESCRIPTION
**Pull request by JuancaG05**
## Related Issues
App: https://github.com/owncloud/android/issues/4397

- [x] Add changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
- [x] Add feature to Release Notes in `ReleaseNotesViewModel.kt` creating a new `ReleaseNote()` with String resources (if required)

_____

### CODE REVIEW

File: owncloudApp/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -730,6 +730,8 @@
     <string name="release_notes_4_3_0_subtitle_accessibility_improvements">Some improvements to make the application more accessible</string>
     <string name="release_notes_4_4_0_title_improved_from_original_folder_auto_upload">Auto uploads more consistent</string>
     <string name="release_notes_4_4_0_subtitle_improved_from_original_folder_auto_upload">Added some mechanisms over retries to improve reliability</string>
+    <string name="release_notes_4_4_0_title_resharing_capability">UX improvement in private shares</string>
```
Comment (Aitorbp) [inline owncloudApp/src/main/res/values/strings.xml:733]: I would change `UX` for another name that would be understood by a normal user.

**Review by Aitorbp — CHANGES_REQUESTED** [review]
Just a comment and go

File: owncloudApp/src/main/res/values/strings.xml (hunk 1 above)
Comment (JuancaG05) [reply owncloudApp/src/main/res/values/strings.xml]: Done

**Review by Aitorbp — APPROVED** [review]
LGTM 🥇

**Comment by jesmrec** [comment]
Some checks here:

1. `resharing: true`

- [ ] Sharing file with all permissions -> permissions: 19 in request + all permissions available in UI
- [ ] Sharing folder with all permissions -> permissions: 31 in request + all permissions available in UI

2. `resharing: false`

- [ ] Sharing file with all permissions -> permissions: 3 in request + only edit permission available in UI
- [ ] Sharing folder with all permissions -> permissions: 15 in request + only edit permissions available in UI with subpermissions

**Comment by jesmrec** [comment]
Everything passed. OK from my side

**Pull request merged**